app/src/main/java/com/example/final_project/LoginActivity.java

```diff
@@ -0,0 +1,59 @@
+package com.example.final_project;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Toast;
+import androidx.appcompat.app.AppCompatActivity;
+import com.example.final_project.databinding.ActivityLoginBinding;
+
+/**
+ * This is a simple login activity that allows users to log in with their username and password.
+ * @author Mariah Stinson
+ * <br>
+ * created: 7/30/2025
+ * @since 0.1.0
+ */
+
+public class LoginActivity extends AppCompatActivity {
+    private ActivityLoginBinding binding;
+    //private User user = null;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        binding = ActivityLoginBinding.inflate(getLayoutInflater());
+        setContentView(binding.getRoot());
+
+        binding.loginButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                verifyUser();
+            }
+        });
+    }
+
+    static Intent loginIntentFactory(Context context, String username) {
+        Intent intent = new Intent(context, MainActivity.class);
+        intent.putExtra(MainActivity.MAIN_ACTIVITY_USERNAME_KEY, username);
+        return intent;
+    }
```

Comment:
Just need to update the rest of this method, so we can navigate back to the login page if we need to.
```suggestion
    static Intent loginIntentFactory(Context context) {
        Intent intent = new Intent(context, LoginActivity.class);
        return intent;
    }
```
(I forgot to submit this comment lol)